src/lib/components/Contact.svelte

```diff
@@ -0,0 +1,16 @@
+<script>
+    import BlackButton from './BlackButton.svelte';
```

Comment:
For the sake of future-proofing the component (from being moved around), let's use absolute import paths here relative to `$lib` instead.
```suggestion
    import BlackButton from '$lib/components/BlackButton.svelte';
```